src/database/operations.py

```
@@ -0,0 +1,284 @@
+                "completed_extractions": completed_extractions,
+                "failed_extractions": failed_extractions,
+                "success_rate": round(success_rate, 2),
+                "last_updated": datetime.utcnow().isoformat()
+            }
+
+
+# Combined operations class for convenience
+class BankruptcyDBOperations:
+    """Combined database operations for all bankruptcy data"""
+    
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)